pkg/printers/internalversion/printers.go

```diff
@@ -1673,11 +1673,24 @@ func printEvent(obj *api.Event, options printers.GenerateOptions) ([]metav1.Tabl
 	if obj.FirstTimestamp.IsZero() {
 		firstTimestamp = translateMicroTimestampSince(obj.EventTime)
 	}
+
 	lastTimestamp := translateTimestampSince(obj.LastTimestamp)
 	if obj.LastTimestamp.IsZero() {
 		lastTimestamp = firstTimestamp
 	}
 
+	count := obj.Count
+	if obj.Series != nil {
+		lastTimestamp = translateMicroTimestampSince(obj.Series.LastObservedTime)
+		// Add the first event to the count.
+		count = obj.Series.Count + 1
```

Comment:
If I'm reading the proposal and code correctly, there's no need to add one here.

Reply:
When a series is created for the first time, its [count is set to 1](https://github.com/kubernetes/client-go/blob/v0.19.0-beta.2/tools/events/event_broadcaster.go#L168-L171). However, for a series to be created, there needs to be an isomorphic singleton event created beforehand. This singleton event is not counted in the series count which is why I added one here.